pkgs/misc/emulators/maiko/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, cmake, libX11 }:
+
+stdenv.mkDerivation rec {
+  pname = "maiko";
+  version = "2021-04-14";
```

Comment:
unstable- prefix for non-releases?

Reply:
I don't think that the developers intend to cut releases so adding unstable would be misleading.

Reply:
```suggestion
  version = "unstable-2021-04-14";
```
Still we are fetching a non release version.